src/test/groovy/info/solidsoft/gradle/pitest/WithPitestTaskInitialization.groovy

```diff
@@ -1,14 +1,16 @@
 package info.solidsoft.gradle.pitest
 
+import groovy.transform.CompileDynamic
 import groovy.transform.SelfType
 
 @SelfType(BasicProjectBuilderSpec)
-//@CompileStatic  //Fails with: Error:(13, 16) Groovyc: [Static type checking] - Non static method info.solidsoft.gradle.pitest.BasicProjectBuilderSpec#getJustOnePitestTaskOrFail cannot be called from static context
+@CompileDynamic
```

Comment:
You removed the comment which explains why it cannot be statically compiled (I wonder, if it will work in Groovy 3?)

Reply:
To be precise, It would be good to bring it back :).